src/main/java/net/dv8tion/jda/core/entities/impl/GameImpl.java

```diff
@@ -17,63 +17,11 @@
 
 import net.dv8tion.jda.core.entities.Game;
 
-public class GameImpl implements Game
+@Deprecated
+public class GameImpl extends Game
```

Comment:
Why is this Deprecated instead of removed?
This is an internal class, and as such, we shouldn't be deprecating. It should be removed wholesale.

Reply:
Even as it is a whole class, it was public and removing a public class is a breaking change. I can imagine users that didn't read the docs and used this class directly in setGame methods.

Reply:
It doesn't matter. This is an Impl class, and as such, we don't deprecate them.
Please remove.